fiona/drvsupport.py

```diff
@@ -99,7 +99,7 @@
     # multi-layer
     #   ("OpenAir", "r"),
     # PCI Geomatics Database File 	PCIDSK 	No 	No 	Yes, using internal PCIDSK SDK (from GDAL 1.7.0)
-    ("PCIDSK", "rw"),
+    ("PCIDSK", "raw"),
```

Comment:
@akrherz is this change intended?

Reply:
Yes, it was necessary to make the build work and copied from Toblerity/Fiona#1046

Reply:
This is from https://github.com/Toblerity/Fiona/pull/1046#issuecomment-974906111